src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const { name: commentName, email: commentEmail, body: commentBody } = comment;
```

Comment:
```suggestion
  const { 
    name: commentName, 
    email: commentEmail, 
    body: commentBody,
  } = comment;
```
Also, there is actually no need to change the original names

Reply:
+